src/content/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -351,28 +382,9 @@ You can read, create, update, or delete obfuscation expressions by using the New
 
 ## Obfuscation rules [#obfuscation-rules]
```

Comment:
Do obfuscation rules depend on the creation of expressions? That is, to make a rule, do you always have to create your expressions first? If so, maybe finding a way to make that throughline more obvious somewhere in the doc. (Not necessarily here, but could be good Intro info, if you think it's important enough :) )

Reply:
Disregard! You DO do that in line 10 :joy: